frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/SyncCurrentToParentByMergeAction.java

```diff
@@ -0,0 +1,17 @@
+package com.virtuslab.gitmachete.frontend.actions.traverse;
+
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+import com.virtuslab.gitmachete.frontend.actions.base.BaseSyncToParentByMergeAction;
+import com.virtuslab.gitmachete.frontend.actions.expectedkeys.IExpectsKeySelectedBranchName;
+
+public class SyncCurrentToParentByMergeAction extends BaseSyncToParentByMergeAction
```

Comment:
I wouldn't expose this action too prominently, it's generally discouraged to sync to parent by merge: see https://github.com/VirtusLab/git-machete/#can-i-use-git-merge-for-dealing-with-stacked-prs and https://github.com/VirtusLab/git-machete-intellij-plugin/pull/1115

Reply:
It has a "discouraged" comment under it in the dialogue. Though I can remove it as well. WDYT?

Reply:
it is actually merge parent in branch, rather than the other way around.

Reply:
Yeah, remove completely, I've seen too much human suffering due to merge commits used for stacked branches